frontend/app/login/page.tsx

```diff
@@ -38,14 +38,14 @@ export default function Login() {
           <h1 className="text-2xl font-bold pb-3">Create a new account</h1>
           <p className="pb-3">
             Already made an account?{" "}
-            <span
+            <p
               onClick={() => {
                 changeForm();
               }}
-              className="hover:shadow-md font-bold"
+              className="hover:shadow-lg hover:shadow-white font-bold"
             >
               Login
-            </span>
+            </p>
           </p>
           <RegisterForm />
         </div>
```

Comment:
In Next.js `page.tsx` should be barebones and reserved for server-side rendering. Put the child (client-side) components in the corresponding `components/` folder and put the "use client" directive on them.

Reply:
say no more